docs/reference/class/MongoDBModelDatabaseInfoIterator.txt

```diff
@@ -0,0 +1,22 @@
+.. _php-model-database-info-iterator:
+
+==========================================
+MongoDB\\Model\\DatabaseInfoIterator Class
```

Comment:
I can't say what's responsible for this, but I noticed some of the links under "Enumeration Classes" leave the UI in a strange state. For instance, https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-36027/reference/class/MongoDBModelDatabaseInfoIterator/ scrolls past all of the main content.

This could be due to the long titles. In this case, I'd be amenable to stripping the full namespace from the class page titles and preserving the fully qualified class name (FQCN) below in the Definition section.

So this can just be "DatabaseInfoIterator Class" if that helps with navigation. I think that would still be fine for SEO purposes.

Reply:
Seems like shortening the title fixed the scrolling issue

Reply:
This does still happen with some other pages: https://preview-mongodbnorareidy.gatsbyjs.io/php-library/DOCSP-36027/reference/method/MongoDBModelCollectionInfo-isCapped/

I imagine this isn't specific to PHPLIB, though, and is probably a more general issue with the docs theme. Perhaps worthy of a DOCSP ticket to be addressed later.

Reply:
Ah, yeah I think this issue has come up before. I'll follow up with Docs platform.